index.html

```diff
@@ -607,9 +607,10 @@ <h2>
         </dt>
         <dd>
           <p>
-            Diversity for the purposes of diversity and inclusion, is any
-            dimension that can be used to differentiate groups and people from
-            one another. Dimensions can include but are not limited to:  socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.
+            Diversity is variation across a wide range of identities, lived experiences, abilities, and perspectives.
+          </p>
+          <p>
+            This can include, but is not limited to: socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, <a href="https://www.google.com/search?q=indigeneity+definition&oq=indi&aqs=chrome.0.69i59l3j0i433i512j69i57j46i131i433i512j46i131i199i433i465i512j0i512j46i131i433i512j0i433i512.2396j0j7&sourceid=chrome&ie=UTF-8">indigeneity</a>, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.
```

Comment:
We agreed in today's meeting to change the URL to the one Chris suggested https://www.google.com/search?q=indigeneity

Reply:
I don't want to hold up publication, so I suggest we go ahead and merge using the google link as decided on today's call.  But I think it would be good to improve this in the future -- next year? -- and I think the research I turned up qualifies as sufficient new information to re-open the topic for consideration in a future revision of the document.  I can open it as a new issue after you have merged, and note in the issue that it is not intended for immediate consideration.